.changeset/calm-years-repeat.md

```diff
@@ -0,0 +1,5 @@
+---
+'highlight.run': patch
+---
+
+fix otel webjs startSpan crashing when otel code has not loaded
```

Comment:
@Vadman97 do we need to bump the version number for this? Also, do we need to add an entry in the changelog?

Reply:
yep, that's what the `yarn changeset version` command will do with this file. normally i release the SDK in a separate PR, but you're right that I should do that here so we can get the SDK change out asap.